Makefile

```diff
@@ -87,7 +88,7 @@ dockers_dev: $(DOCKERS_DEV)
 
 define docker_push
 	for svc in $(SERVICES); do \
-		docker push mainflux/$$svc:$(1); \
+		docker push $(DOCKER_IMAGE_NAME_PREFIX)mainflux/$$svc:$(1); \
```

Comment:
I think that we can use `$(svc)` here

Reply:
Without `$(MF_DOCKER_IMAGE_NAME_PREFIX)` there will be just service name, so for docker it would be different images

Reply:
I'm maybe wrong but my question was about `$$svc`

Reply:
I assume that is needed because there is used `shell for loop`, and to access shell variables we should use `$$` instead of `$`

Reply:
right